cmd/tempo-cli/main.go

```diff
@@ -131,6 +133,16 @@ func loadBackend(b *backendOptions, g *globalOptions) (backend.Reader, backend.W
 		cfg.StorageConfig.Trace.Azure.ContainerName = b.Bucket
 	}
 
+	cfg.StorageConfig.Trace.S3.InsecureSkipVerify = b.InsecureSkipVerify
```

Comment:
let's set the same field on the GCS config so it applies to both backends.

`cfg.StorageConfig.Trace.GCS.Insecure`

It seems like we don't have a similar option on Azure. Can we mention that in the `help` text? something like "only applies to S3 and GCS"